xbmc/GUIInfoManager.cpp

```diff
@@ -1728,6 +1728,12 @@ const infomap weather[] =        {{ "isfetched",        WEATHER_IS_FETCHED },
 ///     @skinning_v19 **[New Boolean Condition]** \link  System_SupportsCPUUsage `
 ///     System.SupportsCPUUsage`\endlink <p>
 ///   }
+///   \table_row3{   <b>`System.SupportedHDRTypes`</b>,
+///                  \anchor System_SupportedHDRTypes
+///                  _string_,
+///     @return The display's supported HDR types.
```

Comment:
misses skinning api revision like the others above:
```
///     <p><hr>
///     @skinning_v20 **[New Infolabel]** \link  System_SupportedHDRTypes `
```

Reply:
Thanks.